musica/micm/micm.meta

```diff
@@ -45,19 +51,19 @@
   type = real | kind = kind_phys
   dimensions = (horizontal_loop_extent,vertical_layer_dimension)
   intent = in
-[ time_step ]
-  standard_name = timestep_for_physics
-  units = s
+[ dry_air_density ]
+  standard_name = density_of_dry_air
```

Comment:
This should be changed to "dry_air_density" (based off conversations in my current standard names PR):

```suggestion
  standard_name = dry_air_density
```

Reply:
Done

Reply:
Hi @nusbaume, I learned that this PR has not been merged yet. I guess we could wait for your PR merged in because if not, CAM-SIMA would complain about `dry_air_density`.  I also think  `NamesNotInDictionary.txt` has to be regenerated. Could you confirm it?